src/openforms/formio/migration_converters.py

```diff
@@ -45,10 +45,50 @@ def move_time_validators(component: Component) -> bool:
     return True
 
 
-# Keys are component["type"] values, values are dicts keyed by unique converter
-# identifier and the function to do the actual conversion.
+def alter_prefill_default_values(component: Component) -> bool:
+    """A converter that replaces ``prefill`` dict values from ``None`` to an empty string."""
+    if "prefill" not in component:
+        return False
+
+    altered = False
+
+    prefill_plugin = component["prefill"]["plugin"]
+    prefill_attribute = component["prefill"]["attribute"]
```

Comment:
there's a chance that `prefill` is present, but doesn't have the keys `plugin` or `attribute` (or is even `None`), so I'd be a bit more defensive here:

```py
unset = object()

if not (prefill := component.get("prefill") or {}):
    return False

prefill_plugin = prefill.get("plugin", unset)
if prefill_plugin is None:
    ... etc.
```

Reply:
Added tests for this as well